pom.xml

```diff
@@ -129,6 +129,7 @@
                             <excludes>
                                 <exclude>*</exclude>
                                 <exclude>codestyle/**</exclude>
+                                <exclude>uat/*</exclude>
```

Comment:
I don't think we should exclude the licensing headers from the uat folder.

Reply:
It should be OK, because it only exclude the files right under under uat folder, i.e. for now only the readme.md and pom.xml is excluded. other files not excluded.